frontend/lib/norent/site.tsx

```diff
@@ -20,6 +20,16 @@ import {
 } from "./letter-content";
 import Navbar from "../ui/navbar";
 import { createLetterStaticPageRoutes } from "../static-page/routes";
+import { NorentFaqsPage } from "./faqs";
+import { NorentInfoPage } from "./info";
+import { NorentAboutYourLetterPage } from "./about-your-letter";
+
+const ROUTES_FOR_PRIMARY_PAGES = [
+  Routes.locale.home,
+  Routes.locale.info,
+  Routes.locale.faqs,
+  Routes.locale.aboutLetter,
+];
```

Comment:
Urp, so one thing about `Routes` is that you can't actually access `Routes.locale` in "top-level code", because the i18n system (which defines exactly what locale prefix to use) might not be initialized yet--or even if it is, on the s server-side it could change depending on what user we're rendering HTML for.  So you should change this to `getRoutesForPrimaryPages()` and just have it return this array dynamically--that will ensure that whenever it's called, it will always return the proper locale-prefixed routes for the current user.

Reply:
Another _extremely_ minor thing is that since you seem to only be using this to detect whether a route is in this array, you might want to consider using a [`Set`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) instead.  Sets will have constant lookup time, while lists will be O(n) because the computer will be iterating through each item in the list for comparison.  Since `n` is super small this won't ever be an issue, which is why it's an extremely minor thing, but I thought I'd mention it for future situations where that `n` might be bigger.

Reply:
Cool! Never worked with Set's before, I just tried it out.